backend/src/core/utils/rds-iam.ts

```diff
@@ -0,0 +1,37 @@
+import AWS from 'aws-sdk'
+import { Config } from 'sequelize'
+
+import config from '@core/config'
+import { configureEndpoint } from '@core/utils/aws-endpoint'
+
+const rdsSigner = new AWS.RDS.Signer(configureEndpoint(config))
+
+export type MutableConfig = {
+  -readonly [K in keyof Config]: Config[K]
```

Comment:
Hmmm whats the goal for using -readonly?

Reply:
The `password` field in the exported `Config` type from `sequelize` is [by default readonly](https://github.com/sequelize/sequelize/blob/81c33d8c2fb719409bf5692eb6a50a5883c2b6f3/types/lib/sequelize.d.ts#L144). However, we need to set it to the generated auth token in the `beforeConnect` hook. Hence, I decided to recast `Config` to a custom `MutableConfig` type where all fields are mutable. Alternatively, we could cast to `any` but decided to do this to preserve the type information.